### ISSUE TITLE
error in generate_tfrecord.py 

### ISSUE THREAD

**Issue by macgihan** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#347]
i have two lables  one is "Healthy" and other is "Infected"

# TO-DO replace this with label map
def class_text_to_int(row_label):
    if row_label == 'Healthy':
        return 1
    elif row_label == 'Infected':
        return 2
    else:
        None

but when i run it i get error like this  please help 
---------------------------------------------------------------------------------------
file "generate_tfrecord.py", line 102, in <module> tf.app.run() file "c:\users\gamag\anaconda3\envs\tensorflow2\lib\site-packages\tensorflow\python\platform\app.py", line 40, in run _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef) file "c:\users\gamag\anaconda3\envs\tensorflow2\lib\site-packages\absl\app.py", line 299, in run _run_main(main, args) file "c:\users\gamag\anaconda3\envs\tensorflow2\lib\site-packages\absl\app.py", line 250, in _run_main sys.exit(main(argv)) file "generate_tfrecord.py", line 93, in main tf_example = create_tf_example(group, path) file "generate_tfrecord.py", line 82, in create_tf_example 'image/object/class/label': dataset_util.int64_list_feature(classes), file "c:\tensorflow1\models\research\object_detection\utils\dataset_util.py", line 30, in int64_list_feature return tf.train.feature(int64_list=tf.train.int64list(value=value)) typeerror: none has type nonetype, but expected one of: int, long
------------------------------------------------------------------------------------------------------

**Comment by macgihan** [comment]
fixed by removing 
"return non"-----> return 0
